engine/engine.go

```diff
@@ -423,6 +429,12 @@ func (e *Engine) Start() error {
 		go WebsocketRoutine()
 	}
 
+	if e.Settings.EnableGCTScriptManager {
+		if err := e.GctScriptManager.Start(); err != nil {
+			log.Errorf(log.Global, "GCTScript manager unable to start: %v", err)
```

Comment:
Just noticed that we have a consistency issue with these (and in the order manager below also), on some we mention the sublogger and on others we use log.Global. I think it's best if we use the specific sublogger as it could help with filtering events based off a subsystem

Reply:
I have been going through and cleaning up a lot of these today trying to distinguish between using global v sublogger

My general thought train is if there is a chance it might get logged _before_ or _after_ a subsystem is enabled/disabled keep it as global if it happens during execution from a subsystem leave it as that 

In this example GCTScript has failed to start and I kept it as global 

are you happy with this path or would you rather me just always go with the subsystem its referring to?

Reply:
I like that train of thought (use global if before or after a subsystem is enabled/disabled), otherwise use the actual subsystem sublogger

Reply:
I have made this change everything that starts before script manager (such as start/stop messages) now use global while everything from within the subsystem use gctscriptmgr